index.js

```diff
@@ -385,6 +387,23 @@ class Bot extends EventEmitter {
     }
 
     /**
+     * Remove @ or # character from group | channel | user name
+     * @param {string} name
+     * @returns {string}
+     */
+    _cleanName (name) {
+        if (name === undefined || name.lenght === 0) {
```

Comment:
Probably, you can check that an argument is exist and it is a string `(name && typeof name === 'string')`